src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,108 @@
+package duke;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Parser {
+    enum Commands {
+        LIST("list"),
+        MARK("mark"),
+        UNMARK("unmark"),
+        DELETE("delete"),
+        TODO("todo"),
+        DEADLINE("deadline"),
+        EVENT("event"),
+        BYE("bye");
+
+
+        final String command;
+
+        Commands(String command) {
+            this.command = command;
+        }
+    }
+
+    protected static void start() throws DukeException, IOException {
+        //to read user input
+        Scanner scanned = new Scanner(System.in);
+        System.out.println("");
+        String input = scanned.nextLine();
+
+        while (!input.equals(Commands.BYE.command)) {
+            System.out.println("=======================================================================");
```

Comment:
May this line divider can be declared as a constant static field in UI class and used here instead ?